packages/itwin/tree-widget/src/components/trees/stateless/common/components/BaseTree.tsx

```diff
@@ -35,6 +36,7 @@ type TreeRendererProps = Pick<
   | "getHierarchyLevelDetails"
   | "size"
   | "getIcon"
+  | "getLabel"
```

Comment:
Do we need to allow passing `getLabel` callback to `BaseTree`? Looks like `BaseTreeRenderer` overrides it anyway.

Reply:
No, changed.